app/pages/products/hosting-web/hosting-web.ts

```diff
@@ -25,14 +31,30 @@ export class HostingWebPage extends ProductCore {
     this.analytics.trackView('product:hosting-web');
 
     this.subscription = this.hostingWebService.getAll(this.serviceName)
-      .finally(() => this.loading = false)
+      .finally(() => this.loading.init = false)
       .subscribe(
         (server) => this.server = server,
         (err) => {
           this.error = true;
           this.toast.error('Une erreur est survenue lors du chargement : ' + JSON.parse(err._body).message).present();
         }
       );
+
+    this.getChart(this.monitoring, this.monitoringPeriod);
+  }
+
+  getChart(type: string, period: string) {
+    this.loading.monitoring = true;
+    this.hostingWebService.getChart(this.serviceName, type, period)
+      .subscribe(
+        (stats) => this.stats = stats,
+        (err) => this.toast.error('Une erreur est survenue lors du chargement des statistiques : ' + JSON.parse(err._body).message).present(),
+        () => this.loading.monitoring = false
```

Comment:
could you use ` .finally(() => this.loading.monitoring = false) ` instead the third argument please ? :)